mediapipe/tasks/ios/genai/inference/sources/LlmInference+Session.swift

```diff
@@ -159,20 +159,26 @@ extension LlmInference {
       try llmInference.shouldContinueWithResponseGeneration()
 
       /// Used to make a decision about whitespace stripping.
-      var receivedFirstToken = true
+      var receivedFirstNonEmptyToken = false
 
       llmSessionRunner.predictAsync(
         progress: { partialResponseStrings, error in
           guard let responseStrings = partialResponseStrings,
             let humanReadableLlmResponse = Session.humanReadableString(
-              llmResponses: responseStrings, stripLeadingWhitespaces: receivedFirstToken)
+              llmResponses: responseStrings, stripLeadingWhitespaces: !receivedFirstNonEmptyToken)
           else {
```

Comment:
@yishuangP I have changed the name of the state variable and the values it takes in different scenarios to map closely to the  meaning of the  variable name for better readability. So the condition check for leading white space stripping is flipped. The logic remains the same as the previous implementation.